cdap-docs/developer-manual/source/cloud-runtimes/provisioners/remote-hadoop.rst

```diff
@@ -8,5 +8,28 @@
 Remote Hadoop
 =============
 
-Describe what the remote hadoop provisioner is for and each config setting
+The Remote Hadoop provisioner is used to run jobs on a pre-existing Hadoop
+cluster. It does not actually create or delete a cluster. It simply connects
+to one. It is assumed that the cluster provided will be one with compatible
+versions of all software that CDAP requires. The cluster must not use Kerberos
+authentication, and it must be accessible via ssh and https (ports 22 and 443).
 
+The configuration for the Remote Hadoop provisioner is simply the information
+required to SSH to an edge node of the cluster.
+
+Host
+----
+The IP address or hostname of the edge node that CDAP will SSH to when starting
+the pipeline run. The node must be able to submit YARN jobs.
+
+User
+----
+The user to SSH as.
+
+SSH Key
+-------
```

Comment:
The SSH private key to use when connecting to the cluster. Since your SSH key is sensitive, we recommended that you provide the key through the CDAP :ref:`Secure Storage <http-restful-api-secure-storage>` API by adding a secure key with the RESTful API and clicking the shield icon in the UI to select a secure key.